diff --git a/src/http/modules/ngx_http_autoindex_module.c b/src/http/modules/ngx_http_autoindex_module.c
index 450a48e..bdf7c11 100644
--- a/src/http/modules/ngx_http_autoindex_module.c
+++ b/src/http/modules/ngx_http_autoindex_module.c
@@ -489,7 +489,8 @@ ngx_http_autoindex_handler(ngx_http_request_t *r)
             }
 
             b->last = ngx_cpymem(b->last, "</a>", sizeof("</a>") - 1);
-            ngx_memset(b->last, ' ', NGX_HTTP_AUTOINDEX_NAME_LEN - len);
+            if (len < NGX_HTTP_AUTOINDEX_NAME_LEN)
+              ngx_memset(b->last, ' ', NGX_HTTP_AUTOINDEX_NAME_LEN - len);
             b->last += NGX_HTTP_AUTOINDEX_NAME_LEN - len;
         }
 
